.github/workflows/ci.yml

```diff
@@ -88,9 +89,13 @@ jobs:
       - name: Publish images for cache
         if: github.repository_owner == 'getsentry'
         run: |
+          # Useful to speed up CI
           docker push ghcr.io/getsentry/snuba-ci:${{ steps.branch.outputs.branch }}
           docker push ghcr.io/getsentry/snuba-ci:${{ github.sha }}
-          docker push ghcr.io/getsentry/snuba-ci:latest
+          if [ "${{ steps.branch.outputs.branch }}" == 'master' ]; then
```

Comment:
This prevents a PR affecting the execution of another PR or a CI run on `master`.